onnx/custom_element_types.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+"""This module defines custom dtypes not supported by numpy.
```

Comment:
As discussed, I suggest adding the following disclaimer below:
```
The numpy representation dtypes used below are meant for internal use. They may change in the
future based on the industry standardization of these numpy types.
```

I wonder if we should name the file `_custom_element_types.py` perhaps?

Reply:
It is not really internal, it is used by the reference implementation as well.

Reply:
`reference implementation` = internal so I think it's ok. If we are talking about the output values by the reference implementation which takes these types, then I suggest we convey a message of "don't care what the type is except for knowing what the byte representation is."